x-pack/test/functional/apps/code/explore_repository.ts

```diff
@@ -141,6 +145,19 @@ export default function exploreRepositoryFunctonalTests({
         log.info('src folder closed');
       });
 
+      it('highlight only one symbol', async () => {
+        await testSubjects.click('codeFileTreeNode-Directory-src');
```

Comment:
Also, for each click action, I would suggest to first check if the selector exists or not, and then click the time. Otherwise, it test could be very flaky if the click has been issued before the tree has been expanded.